examples/discovery-search-app/cypress/support/index.js

```diff
@@ -14,7 +14,10 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import './commands';
+// import './commands.ts';
+
+// import the default settings for cy.server()
+import './serverDefaults.ts';
```

Comment:
I think if you want to write support files in typescript you need to follow this: 

https://docs.cypress.io/guides/tooling/typescript-support.html#Examples-1
https://github.com/omerose/cypress-support
https://github.com/omerose/cypress-support/blob/master/cypress/plugins/cy-ts-preprocessor.js

Reply:
@maniax89 Do you think this is worth doing as part of this PR?

Reply:
yeah if we need to pull in https://github.com/omerose/cypress-support/blob/master/cypress/plugins/cy-ts-preprocessor.js to get typescript working i guess we could save it for another PR

Reply:
That's my thought as well. And this is just to get typescript on the support files. It still works in the test files without this setup.

Still worth doing at some point, but I don't think we have to hold up this PR.